apps/jobs/views.py

```diff
@@ -539,3 +540,38 @@ def users(self, request):
         return response.create_response(
             serialized_company_data.data, status.HTTP_200_OK
         )
+
+
+class ContactUsViewSet(viewsets.ModelViewSet):
+    """Company object viewsets
+    API: /api/v/contact-us
+    Database: jobs_contactmessage
```

Comment:
change the db name to `tbl_contact_us` because database tables that are a part of job app. are named in this format only.

Reply:
db name updated